tests/fast-integration/eventing-test/utils.js

```diff
@@ -44,15 +42,13 @@ const kymaStreamName = 'sap';
 const isSKR = process.env.KYMA_TYPE === 'SKR';
 const skrInstanceId = process.env.INSTANCE_ID || '';
 const testCompassFlow = process.env.TEST_COMPASS_FLOW || false;
-const testSubscriptionV1Alpha2 = process.env.ENABLE_SUBSCRIPTION_V1_ALPHA2 === 'true';
-const subCRDVersion = testSubscriptionV1Alpha2 ? 'v1alpha2' : 'v1alpha1';
 const isUpgradeJob = process.env.EVENTING_UPGRADE_JOB || false;
+const isUpgradeJob2ndReconcile = process.env.EVENTING_UPGRADE_2ND_RECONCILE_JOB || false;
```

Comment:
In the upgrade job, we are doing two reconciliations,
- One for latest release to PR-xxx version.
- Second for save version i.e. PR-xxx to PR-xxx.

The `EVENTING_UPGRADE_2ND_RECONCILE_JOB` is to identify if this is the second phase of reconciliations, so we can skip the atleast once test, because it was already tested in First reconciliation.